app/services/boarding_service.rb

```diff
@@ -24,7 +24,7 @@ def initialize(app_id: ENV["ITC_APP_ID"],
     @app_id = app_id
```

Comment:
`tester_groups: ENV["ITC_APP_TESTER_GROUPS"].to_s.split(/\s*,\s*/))`

It's probably easier to read if you move that into the method body instead of the parameter declaration